.github/workflows/build_base_on_forks.yml

```diff
@@ -0,0 +1,33 @@
+---
+# This workflow is required to experiment with container images on forks
+# it will NEVER run on aristanetworks/avd and requires following env variables:
+#     DEBUG_CONTAINER_ACTIONS = true
+# The workflow supposed to be the exact replica of container_build_base.yml in a stable state
+# but diff is allowed to support a longer experimental setup across different forks
+name: Build base on forks
+
+"on":
+  push:
+    branches: [ '**' ]
```

Comment:
I don't think this is needed at all.
```suggestion
```

Reply:
@ClausHolbechArista I ended with removing on_fork workflows completely. Looks like it's too much redundancy for little value. This was replaced with `if: ( github.repository == 'aristanetworks/avd' && github.ref == 'refs/heads/devel' ) || ${{ vars.DEBUG_CONTAINER_ACTIONS }} == "true"` for every parent container build workflow. I've added `branches: [ '**' ]` to every on.push however. If you prefer - we can completely remove this line as it's definitely redundant. There are few reasons I've included this line:
- be explicit
- easy to adjust for debug, troubleshooting, etc.
- match workflow_dispatch style
Let me know if you want to remove it anyway and I'll respect your choice.